<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KnimeExportDialog</class>
 <widget class="QDialog" name="KnimeExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="generalBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>528</width>
        <height>283</height>
       </rect>
      </property>
      <attribute name="label">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="exportPath">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="choosePath">
         <property name="text">
          <string>Choose File</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="includeRaw">
         <property name="text">
          <string>Include whole raw image instead of rois</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layerBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>528</width>
        <height>283</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Layers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select object layers to export</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="objLayerView">
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="featureBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>528</width>
        <height>283</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Features</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QPushButton" name="selectAllFeatures">
         <property name="text">
          <string>All</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="selectNoFeatures">
         <property name="text">
          <string>None</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTreeWidget" name="featureView">
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KnimeExportDialog</receiver>
   <slot>validate_before_exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KnimeExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectAllFeatures</sender>
   <signal>clicked()</signal>
   <receiver>KnimeExportDialog</receiver>
   <slot>select_all_features()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectNoFeatures</sender>
   <signal>clicked()</signal>
   <receiver>KnimeExportDialog</receiver>
   <slot>select_no_features()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>choosePath</sender>
   <signal>clicked()</signal>
   <receiver>KnimeExportDialog</receiver>
   <slot>choose_path()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>includeRaw</sender>
   <signal>stateChanged(int)</signal>
   <receiver>KnimeExportDialog</receiver>
   <slot>include_raw_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select_all_features()</slot>
  <slot>select_no_features()</slot>
  <slot>validate_before_exit()</slot>
  <slot>choose_path()</slot>
  <slot>include_raw_changed(int)</slot>
 </slots>
</ui>
